src/assets/styles/10-settings/_spacing.scss

```diff
@@ -0,0 +1,48 @@
+// ----------------------------------
+// UI Settings
+// Spacing
+// https://zpl.io/bJWJNgW
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+:root {
+  --space-unit: 1rem;
+
+  // --space-xs: 4px
+  // --space-sm: 6px
+  // --space-md: 8px
+  // --space-lg: 16px
+  // --space-xl: 32px
+  --space-xs: calc(0.25 * var(--space-unit));
+  --space-sm: calc(0.375 * var(--space-unit));
+  --space-md: calc(0.5 * var(--space-unit));
+  --space-lg: calc(1 * var(--space-unit));
+  --space-xl: calc(2 * var(--space-unit));
+
+  // --space-sm: 8px
+  // --space-md: 12px
+  // --space-lg: 24px
+  // --space-xl: 48px
+  @include breakpoint(sm) {
+    --space-sm: calc(0.5 * var(--space-unit));
+    --space-md: calc(0.75 * var(--space-unit));
+    --space-lg: calc(1.5 * var(--space-unit));
+    --space-xl: calc(3 * var(--space-unit));
+  }
+
+  // --space-md: 16px
```

Comment:
Do you think we should keep the comments detailing pixel values? Means we have to update them if/when the sizes change (although I'm not sure they will change much, if at all...)

Reply:
It's good for quick reference, esp if it means not opening Zeplin each time

They should be updated at the same time as the actual values